terraform/pubsub.tf

```diff
@@ -245,6 +245,28 @@ resource "google_pubsub_subscription" "default" {
   }
 }
 
+resource "google_pubsub_subscription" "json" {
```

Comment:
Im confused how this is different from the current default subscription?

Reply:
Its not. I want two identical subscriptions to the same topic. I plan to dual-write and then later turn off the old stream once everything looks good.

Reply:
Ah knowing that you cant just add a partition, this makes sense now, thanks!!